Xenon/Xe_Main.cpp

```diff
@@ -110,12 +116,15 @@ int main(int argc, char* argv[])
 	pciBridge.addPCIDevice(&hdd);
 	pciBridge.addPCIDevice(&smc);
 
+	/*****Register the Xenos GPU and the PCi Bridge*****/
 	hostBridge.RegisterXGPU(&xenos);
 	hostBridge.RegisterPCIBridge(&pciBridge);
 
+	/***************************************Initializes the NAND and the RAM***************************************/

```

Comment:
Comments should be the same size regarding the functon/object they describe. And max at line no. 100.